vivisect/__init__.py

```diff
@@ -2118,8 +2118,59 @@ def makePointer(self, va, tova=None, follow=True):
         if follow and self.isValidPointer(tova):
             self.followPointer(tova)
 
+        # Get type of target and hand into addXref()
+        rflags = 0
+        toloc = self.getLocation(tova)
+        if toloc and toloc[L_LTYPE] == LOC_OP:
+            rflags = envi.BR_PROC
+
+        self.addXref(va, tova, REF_PTR, rflags)
+
         return ploc
 
+    def makePointerArray(self, startva, stopva=0, count=0, break_on_bad=True):
```

Comment:
We should use `None` as the default for unused integer arguments to eliminate the possibility of ambiguity (albeit in unlikely edge cases) :)